### PR TITLE
Passing tests

### PR DESCRIPTION
**Pull request by IgorWounds**
Fix crypto, forex, economy, stocks, futures, crypto, news, fixedincome, and charting.

### CODE REVIEW

File: openbb_platform/extensions/economy/integration/test_economy_api.py [hunk 1]
```diff
@@ -469,6 +464,7 @@ def test_economy_gdpforecast(params, headers):
     assert result.status_code == 200
 
 
+@pytest.mark.skip(reason="Not being picked up by the API.")
```
Comment (hjoaquim) [inline openbb_platform/extensions/economy/integration/test_economy_api.py:467]: Should we open a bug with this?

File: openbb_platform/extensions/economy/integration/test_economy_api.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def test_economy_const(params, headers):
     [
         (
             {
-                "countries": ["portugal", "spain"],
+                "countries": "spain",
```
Comment (hjoaquim) [inline openbb_platform/extensions/economy/integration/test_economy_api.py:41]: can we also have a test for multiple countries?

**Review by deeleeramone — APPROVED** [review]

**Review by hjoaquim — CHANGES_REQUESTED** [review]
I'm getting this for economy (all of them fail with http 500):

image

The other tests are good.

File: openbb_platform/extensions/economy/integration/test_economy_api.py (hunk 1 above)
Comment (IgorWounds) [reply openbb_platform/extensions/economy/integration/test_economy_api.py:467]: Probably, yes.

Comment (hjoaquim) [reply openbb_platform/extensions/economy/integration/test_economy_api.py:467]: Done.

**Comment by IgorWounds** [comment]
> I'm getting this for economy (all of them fail with http 500):
> 
> image
> 
> The other tests are good.

Can you try again and check your credentials? @hjoaquim 

Screenshot 2023-10-13 at 19 47 16

**Review by hjoaquim — APPROVED** [review]
I'm getting this for economy:

**Pull request merged**